Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/MainActivity.java

```diff
@@ -113,8 +114,16 @@ private void analyze() {
     totalFrames =
         Integer.valueOf(mediaMetadataRetriever.extractMetadata(METADATA_KEY_VIDEO_FRAME_COUNT));
     // TODO: Add on demand frame read opposed to read all frames.
-    frameList = mediaMetadataRetriever.getFramesAtIndex(0, Math.max(0, totalFrames - SAFE_FRAMES));
+    for (int i = 0; i < totalFrames; i += FRAME_CHUNK_READ_SIZE) {
+      try{
+        frameList.addAll(mediaMetadataRetriever.getFramesAtIndex(
+            i, Math.min(totalFrames - i, FRAME_CHUNK_READ_SIZE)));
+      }catch (IllegalStateException e){
```

Comment:
Please run the formatter to fix up the whitespace before submitting.
https://google.github.io/styleguide/javaguide.html#s4-formatting
try {
...
} catch (IllegalStateException e) {
 ...
}

Reply:
As discussed, I'll merge now and we'll fix this in a follow up.